optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticType.java

```diff
@@ -23,4 +23,9 @@
      */
     public String name();
 
+    /**
+     * @return never null
+     */
+    public String formattedName();
```

Comment:
Good proposals. I'm not really sure which one we should choose.

A) I agree, but we should consider keeping `name()` - as our enum overrides it by itself IIRC: http://docs.oracle.com/javase/7/docs/api/java/lang/Enum.html#name%28%29

Else `getName()` would basically just `return name()`

B) This proposal is intriguing, but I don't really like the idea that we would localize something we use for folder/file names. I know we already support it for benchmark names, but doesn't feel right either.